apps/oxlint/src/snapshots/fixtures__config_ignore_patterns__with_oxlintrc_-c .__test__eslintrc.json --ignore-pattern _.ts .@oxlint.snap

```diff
@@ -5,13 +5,7 @@ source: apps/oxlint/src/tester.rs
 arguments: -c ./test/eslintrc.json --ignore-pattern *.ts .
 working directory: fixtures/config_ignore_patterns/with_oxlintrc
 ----------
-
-  ! ]8;;https://oxc.rs/docs/guide/usage/linter/rules/unicorn/no-empty-file.html\eslint-plugin-unicorn(no-empty-file)]8;;\: Empty files are not allowed.
-   ,-[main.js:1:1]
-   `----
-  help: Delete this file or add some code to it.
```

Comment:
In `eslintrc.json` is `*.js` ignored. This looks like a fix for me.
When someone uses a configuration file in a directory with `-c`, the `ignorePatterns` should not adjust to its directory (`/test/`)